README.md

```diff
@@ -14,19 +14,10 @@ See https://nekovm.org/
 
 ## Snapshot Builds
 
-Compiled binaries can be found in the "artifacts" link in the summary section of each [Azure Pipelines build](https://dev.azure.com/HaxeFoundation/GitHubPublic/_build?definitionId=2&_a=summary&repositoryFilter=2&branchFilter=14&statusFilter=succeeded).
+Compiled binaries can be found in the "artifacts" section in the summary of each [Github Actions build](https://github.com/HaxeFoundation/neko/actions).
```

Comment:
Maybe keep the filters? So `?query=branch%3Amaster+is%3Asuccess` for github actions page